assets/compat/config_spec.yaml

```diff
@@ -115,21 +115,23 @@ bitcoind:
   name: Bitcoin Core
   description: |
     The Bitcoin Core node to connect to:
+      - none: Use the light bitcoin backend built into LND, Neutrino.
       - internal: The Bitcoin Core and Proxy services installed to your Embassy
-      - external: An unpruned Bitcoin Core node running on a different device
   tag:
     id: type
     name: Type
     variant-names:
+      none: None (Built-in LND Neutrino)
       internal: Internal (Bitcoin Core)
```

Comment:
No need for "internal". It's none, core, or proxy

Reply:
I'd recommend against changing the enum variants, but we can change the title renders for sure.

Reply:
will change the copy once we go to master, this PR doesn't have the JS changes

Reply:
Copy change made here: https://github.com/Start9Labs/lnd-wrapper/commit/79e90b8f144d553ac5be90aec566140c6c1940de